### DISCUSSION TITLE
How to: automatically unmount the camera during/after OS start up?

### DISCUSSION THREAD

**Discussion by maehw** [xtech/self-o-mat#25] [Q&A]
I am not a Linux expert. How do you automatically unmount the camera so that the self-o-mat software can access it?

At the moment I have to manually unmount it - as with my previous HW & SW setup. I think there is a way to do it automatically and you're doing it with the sold kits.

@ClemensElflein  Where do I need to dig for a script? `/etc/rc*/`? Or can you guide us directly?

Would be great for automation. I'd like to add the info to the wiki (Configuration page, OS setup section).

**Comment by ClemensElflein** [comment]
In the "official" kits, there simply is no automount script since Raspbian Lite does not ship one. Therefore the problem does not exist there.

In order to prevent automount just remove the SD card from your DSLR. It should work then without a problem.

**Reply by maehw** [reply]
Good idea about removing the SD card from the camera. I was not aware that worked - as I'm quite sure that did not work for my old DSLR (or at least not with an older release of gphoto2).

BTW `gio mount -s gphoto2` should do the trick, also for inserted SD cards (works for me™).
